lib/generator/code_generator.dart

```diff
@@ -61,61 +36,32 @@ class GraphQLCodeGenerators {
       String namespace,
       Map<String, String> fragments,
       Map<String, DartType> types,
-      List<ObjectType> enumObjectTypes,
-      List<ObjectType> interfaceObjectTypes,
-      List<ObjectType> unionObjectTypes,
-      List<ObjectType> objectObjectTypes,
-      List<ObjectType> inputObjectTypes,
+      List<ObjectType> responseTypes,
       ObjectType mutationObjectType) {
     var result = '';
     var emitter = DartEmitter(Allocator());
     Map<String, Class> classes = {};
 
     List<String> enumString =
-        EnumGenerator().enumGenerator(enumObjectTypes, namespace: namespace);
-    Map<String, Class> interfaces = InterfaceGenerator().generate(
-        types,
-        enumObjectTypes,
-        interfaceObjectTypes,
-        unionObjectTypes,
-        objectObjectTypes,
-        inputObjectTypes,
-        namespace: namespace);
+        EnumGenerator().enumGenerator(responseTypes, namespace: namespace);
+    Map<String, Class> interfaces = InterfaceGenerator()
```

Comment:
What about making these `Iterable<Class>` instead of `Map<String,Class>`? Is there an advantage to keeping these as maps?